frontend/dockerfile/linter/docs/LegacyKeyValueFormat.md

```diff
@@ -28,3 +28,33 @@ ARG foo bar
 FROM alpine
 ARG foo=bar
 ```
+
+❌ Bad: multi-line variable declaration with a space separator.
+
+```dockerfile
+ENV DEPS \
+    autoconf \
+    dpkg-dev \
+    file \
+    g++ \
+    gcc \
+    libc-dev \
+    make \
+    pkg-config \
+    re2c
+```
+
+✅ Good: use an equals sign and wrap the value in quotes.
+
+```dockerfile
+ENV DEPS="\
+    autoconf \
+    dpkg-dev \
+    file \
+    g++ \
```

Comment:
nitpick: Do we need so many lines to prove the point? 3-4 would do probably.

Reply:
changed it to `curl git make`